templates/zerver/for/companies.md

```diff
@@ -0,0 +1,68 @@
+Zulip's threading allows asynchronous discussion to happen over chat, which
+completely changes what chat can be used for in a company. Zulip's threading
+allows managers to weigh in on discussions even when they can only check in
+once a day, it allows workers to participate in discussions even when they
+live in different time zones, and it allows neighboring teams to keep tabs
+on conversations that might affect them.
+
+Communication is at the core of the operation of every large business. Many
+senior personnel spend most of their days in meetings, and many decisions
+have to bubble up a management chain because employees on the ground don't
+have enough context. Employers also lose out on remote talent because
+communication with remote team members is too burdensome with their existing
+tools.
+
+No chat product comes even close to Zulip in terms of enabling serious
+discussion, engaging remote participants, spreading knowledge to neighboring
+teams, and making efficient use of managers' time.
+
+If you haven't read [why Zulip](/why-zulip), read that first. We've also
+collected a list of features we think will be of particular interest for
+companies using Zulip.
+
+**On-premise and in the cloud**
+
+High quality export and import tools make it easy to start with Zulip in the
+cloud, knowing that you can move to a on-premise deployment (or back) if
+your budget or security needs change.
+
+**No vendor lock-in**
+
+Zulip's license (Apache 2) and [published code base](github.com/zulip/zulip)
+means you will always be able to run Zulip on your own servers, for free,
+forever.
+
+**Highly customizable**
+
+Zulip has a well-designed API, over 100,000 words of developer
+documentation, over 90% test coverage, and full static typing of our python
+codebase.
```

Comment:
This paragraph is about the codebase, not the API.  I might do something more like:

In addition to Zulip's well-designed API and extensive integration library, Zulip's codebase is easy to get into.  With over 100K words of developer documentation, 93% test coverage, and full static typing of our python codebase with mypy, and world-class tooling, Zulip's codebase is a pleasure to work with.  Just ask any of the hundreds of developers, including high school students, who have contributed significant code to Zulip!